azurelinuxagent/common/protocol/extensions_goal_state_from_extensions_config.py

```diff
@@ -527,7 +527,7 @@ def __parse_and_add_extension_settings(settings_node, name, ext_handler, depends
             ext.dependencyLevel = depends_on_level
             thumbprint = handler_settings.get("protectedSettingsCertThumbprint")
             ext.certificateThumbprint = thumbprint
-            ext_handler.settings.append(ext)
+            extension.settings.append(ext)
```

Comment:
looks like we are using `ext` for settings, can we rename that too (If it's small effort) to avoid confusion.

Reply:
fixed

Reply:
Thanks for the change.